CGPs/cgp-0176.md

```diff
@@ -1,6 +1,6 @@
 ---
 cgp: 176
-title: Celo Camp 2025: Season 0 Proposal
+title: Celo Camp 2025 - Season 0 Proposal

```

Comment:
Just a heads up @ImmanuelSegol . This will be needed to be wrapped in quotes to pass the CI as it includes an illegal operator in the yaml

Reply:
done